application/core/LSYii_Validators.php

```diff
@@ -126,6 +126,27 @@ public function xssFilter($value)
                 'news' => true,
                 )
         );
+
+	$config = $filter->getPurifier()->config;
+        $config->set('HTML.DefinitionID', 'html5-definitions');
+        $def = $config->maybeGetRawHTMLDefinition();
+        $max = $config->get('HTML.MaxImgLength');
```

Comment:
What is this line for?

Reply:
Line accidentally committed. Now removed.